cra-memory-considerate-loading/package.json

```diff
@@ -1,6 +1,7 @@
 {
-  "name": "memory-based-loading",
+  "name": "cra-memory-considerate-loading",
```

Comment:
These changes all LGTM. Thanks for aligning the naming around a common scheme.

Reply:
Yes, thanks for your notice.
I like consistency very much. :)